app/src/main/java/org/oppia/android/app/topic/conceptcard/ConceptCardViewModel.kt

```diff
@@ -22,7 +22,7 @@ class ConceptCardViewModel @Inject constructor(
     processConceptCardLiveData()
   }
 
-  /** Sets the value of skillId and binding. Must be called before setting ViewModel to binding */
+  /** Sets the value of skillId */
```

Comment:
At this point, we can probably just omit the KDoc since it's a trivial setter: https://developer.android.com/kotlin/style-guide#exception_self-explanatory_functions.

Reply:
DONE